tests/default/test.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/sh
 set -eo pipefail
+
+# for integers comparisons: checkCounts <testValue> <expectedValue> <testName>
+checkCounts() {
```

Comment:
I think (not 100%) that snake-case is preferred over camel-case for bash scripts 👍

Reply:
You could be right....  I have Java muscle memory that makes me write functions that way.... I'll check and update if appropriate

Reply:
You are right (according to google)
https://google.github.io/styleguide/shell.xml#Function_Names